src/app/data-planes/components/DataPlaneList.vue

```diff
@@ -23,10 +21,54 @@
     @change="emit('change', $event)"
   >
     <template #toolbar>
-      <slot name="toolbar" />
+      <FilterBar
+        class="data-plane-proxy-filter"
+        :placeholder="`tag: 'kuma.io/service: backend'`"
+        :query="props.query"
+        :fields="{
+          name: { description: 'filter by name or parts of a name' },
+          protocol: { description: 'filter by “kuma.io/protocol” value' },
+          service: { description: 'filter by “kuma.io/service” value' },
+          tag: { description: 'filter by tags (e.g. “tag: version:2”)' },
+          zone: { description: 'filter by “kuma.io/zone” value' },
+        }"
+        @fields-change="emit('fields-change', $event)"
+      />
+
+      <KSelect
+        label="Type"
+        :overlay-label="true"
+        :items="[
+          {
+            label: 'All',
+            value: 'all',
+          },
+          {
+            label: 'Standard',
+            value: 'standard',
+          },
+          {
+            label: 'Builtin',
+            value: 'builtin',
+          },
+          {
+            label: 'Delegated',
+            value: 'delegated',
```

Comment:
In regards to your question on `STANDARD/BUILTIN/DELEGATED`, whatever is decided there we should also use here.

Reply:
I moved the values of the filter and those in the list into the message files so we only need to adjust them there now based on what we decide.

Reply:
👍